src/App.tsx

```diff
@@ -2,13 +2,21 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { PostList } from './components/PostList/PostList';
+
+const preparedPosts = [...posts].map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId) || null,
+  comments: comments.filter(comment => (comment.postId === post.id)) || null,
```

Comment:
filter always return arr (if don't find needed elements return empty arr)

```suggestion
  comments: comments.filter(comment => (comment.postId === post.id)),
```